addons/sms/models/sms_api.py

```diff
@@ -71,6 +67,10 @@ def _get_sms_api_error_messages(self):
         )
         return {
             'unregistered': _("You don't have an eligible IAP account."),
-            'insufficient_credit': ' '.join([_('You don\'t have enough credits on your IAP account.'), buy_credits]),
+            'insufficient_credit': ' '.join([_("You don\'t have enough credits on your IAP account."), buy_credits]),
             'wrong_number_format': _("The number you're trying to reach is not correctly formatted."),
+            'duplicate_message': _("A SMS has bee removed since it was duplicated."),
+            'country_not_supported': _("The destination country is not supported."),
+            'incompatible_content': _("The content of the message violates rules applied by our providers."),
```

Comment:
"violates" -> "does not comply with" ?

Would like to avoid usage of this term.